client/src/main.js

```diff
@@ -22,7 +22,7 @@ import App from './App.vue'
 import router from './router'
 import store from './store'
 import './filters'
-
+import * as nem2Sdk from 'nem2-sdk'
```

Comment:
I dont think we need this for here, we can import directly to file when we need it.

Reply:
removed import * as nem2Sdk from 'nem2-sdk'